src/hotspot/share/runtime/os.hpp

```diff
@@ -1008,7 +1008,7 @@ class os: AllStatic {
 
  public:
   inline static bool platform_print_native_stack(outputStream* st, const void* context,
-                                                 char *buf, int buf_size);
```

Comment:
Small nit, can we make this not a reference, but a pointer, and make it optional and possibly default it to nullptr?

Reply:
Then, we should probably name it `lastpc_ptr`? Would that really be better? I'm not conviced.